packages/abi-gen/templates/TypeScript/partials/method_abi_helper.handlebars

```diff
@@ -1,3 +1,26 @@
+{{^if constant~}}
+{{^if debug~}}
```

Comment:
`if ((NOT constant) AND (NOT debug))`. Because we use all 3 functions `if (debug)`